pkgs/tools/admin/awscli2/default.nix

```diff
@@ -29,13 +29,13 @@ let
 in
 with py.pkgs; buildPythonApplication rec {
   pname = "awscli2";
-  version = "2.0.54"; # N.B: if you change this, change botocore to a matching version too
+  version = "2.0.61"; # N.B: if you change this, change botocore to a matching version too
```

Comment:
Shouldn't botcore also be updated?

Reply:
I think botocore is still on a 1.x.x release ([link here](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/python-modules/botocore/default.nix#L15)) to match the current awscli v1. 

> N.B: if you change this, change botocore to a matching version too

I believe this is a note to change the version in the override at the top, to match what v2 of the aws-cli currently requires ([link](https://github.com/aws/aws-cli/blob/v2/setup.cfg#L7)) :v:

Reply:
Can we maybe clarify this that the overwrite in the top is meant?